library.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
+    def print_longest_crime_audio_book(self) -> str:
```

Comment:
Function description could be added. Also, try making a more generalized function, where input parameters could be audiobook : bool = False, genre : str, search : str. etc.